[Bug]:  debian:11 failed: alien --scripts --to-deb  oceanbase-diagnostic-tool*.rpm

### Describe the bug

 debian:11

```bash
alien --scripts --to-deb  oceanbase-diagnostic-tool*.rpm

perl: warning: Falling back to the standard locale ("C").
dh_usrlocal: error: debian/oceanbase-diagnostic-tool/usr/local/oceanbase-diagnostic-tool/init.sh is not a directory
make: *** [debian/rules:7: binary] Error 25
```

### Environment

 debian:11

### Fast reproduce steps

 debian:11

alien --scripts --to-deb  oceanbase-diagnostic-tool*.rpm

### Expected behavior

_No response_

### Actual behavior

_No response_

### Additional context

_No response_

--- a/README-CN.md
+++ b/README-CN.md
@@ -37,7 +37,7 @@ OceanBase Diagnostic Tool (obdiag) 是一款专门为OceanBase打造的敏捷诊
 sudo yum install -y yum-utils
 sudo yum-config-manager --add-repo https://mirrors.aliyun.com/oceanbase/OceanBase.repo
 sudo yum install -y oceanbase-diagnostic-tool
-sh /usr/local/oceanbase-diagnostic-tool/init.sh
+sh /opt/oceanbase-diagnostic-tool/init.sh
 ```
 
 ## 方式二：源码安装

--- a/README.md
+++ b/README.md
@@ -37,7 +37,7 @@ You can install obdiag by using these methods:
 sudo yum install -y yum-utils
 sudo yum-config-manager --add-repo https://mirrors.aliyun.com/oceanbase/OceanBase.repo
 sudo yum install -y oceanbase-diagnostic-tool
-sh /usr/local/oceanbase-diagnostic-tool/init.sh
+sh /opt/oceanbase-diagnostic-tool/init.sh
 ```
 
 ## Method 2: Install obdiag by using the source code

--- a/rpm/init.sh
+++ b/rpm/init.sh
@@ -50,7 +50,7 @@ source  ${WORK_DIR}/init_obdiag_cmd.sh
 
 cd -
 output_file=${OBDIAG_HOME}/version.yaml
-version_line=$(/usr/local/oceanbase-diagnostic-tool/obdiag --version 2>&1 | grep -oP 'OceanBase Diagnostic Tool: \K[\d.]+')
+version_line=$(/opt/oceanbase-diagnostic-tool/obdiag --version 2>&1 | grep -oP 'OceanBase Diagnostic Tool: \K[\d.]+')
 if [ -n "$version_line" ]; then
     content="obdiag_version: \"$version_line\""
 

--- a/rpm/oceanbase-diagnostic-tool.spec
+++ b/rpm/oceanbase-diagnostic-tool.spec
@@ -33,7 +33,7 @@ mkdir -p $BUILD_DIR/SOURCES/site-packages
 mkdir -p $BUILD_DIR/SOURCES/resources
 mkdir -p $BUILD_DIR/SOURCES/dependencies/bin
 mkdir -p ${RPM_BUILD_ROOT}/usr/bin
-mkdir -p ${RPM_BUILD_ROOT}/usr/local/oceanbase-diagnostic-tool
+mkdir -p ${RPM_BUILD_ROOT}/opt/oceanbase-diagnostic-tool
 cp -rf $SRC_DIR/src $BUILD_DIR/SOURCES/site-packages/
 pyinstaller --hidden-import=decimal -p $BUILD_DIR/SOURCES/site-packages -F src/obdiag.py
 rm -f obdiag.py oceanbase-diagnostic-tool.spec
@@ -47,33 +47,33 @@ cd $SRC_DIR
 \cp -rf $SRC_DIR/rpm/init_obdiag_cmd.sh $BUILD_DIR/SOURCES/init_obdiag_cmd.sh
 \cp -rf $SRC_DIR/rpm/obdiag_backup.sh $BUILD_DIR/SOURCES/obdiag_backup.sh
 \cp -rf $SRC_DIR/conf $BUILD_DIR/SOURCES/
-mkdir -p ${RPM_BUILD_ROOT}/usr/local/oceanbase-diagnostic-tool/lib/
-mkdir -p ${RPM_BUILD_ROOT}/usr/local/oceanbase-diagnostic-tool/dependencies/bin
+mkdir -p ${RPM_BUILD_ROOT}/opt/oceanbase-diagnostic-tool/lib/
+mkdir -p ${RPM_BUILD_ROOT}/opt/oceanbase-diagnostic-tool/dependencies/bin
 find $SRC_DIR -name "obdiag"
-\cp -rf $SRC_DIR/dist/obdiag ${RPM_BUILD_ROOT}/usr/local/oceanbase-diagnostic-tool/obdiag
-\cp -rf $BUILD_DIR/SOURCES/site-packages ${RPM_BUILD_ROOT}/usr/local/oceanbase-diagnostic-tool/lib/site-packages
-\cp -rf $BUILD_DIR/SOURCES/resources ${RPM_BUILD_ROOT}/usr/local/oceanbase-diagnostic-tool/resources
-\cp -rf $BUILD_DIR/SOURCES/dependencies/bin ${RPM_BUILD_ROOT}/usr/local/oceanbase-diagnostic-tool/dependencies
-\cp -rf $BUILD_DIR/SOURCES/example ${RPM_BUILD_ROOT}/usr/local/oceanbase-diagnostic-tool/
-\cp -rf $BUILD_DIR/SOURCES/conf ${RPM_BUILD_ROOT}/usr/local/oceanbase-diagnostic-tool/conf
-\cp -rf $BUILD_DIR/SOURCES/init.sh ${RPM_BUILD_ROOT}/usr/local/oceanbase-diagnostic-tool/
-\cp -rf $BUILD_DIR/SOURCES/init_obdiag_cmd.sh ${RPM_BUILD_ROOT}/usr/local/oceanbase-diagnostic-tool/
-\cp -rf $BUILD_DIR/SOURCES/obdiag_backup.sh ${RPM_BUILD_ROOT}/usr/local/oceanbase-diagnostic-tool/
-\cp -rf $BUILD_DIR/SOURCES/plugins ${RPM_BUILD_ROOT}/usr/local/oceanbase-diagnostic-tool/
+\cp -rf $SRC_DIR/dist/obdiag ${RPM_BUILD_ROOT}/opt/oceanbase-diagnostic-tool/obdiag
+\cp -rf $BUILD_DIR/SOURCES/site-packages ${RPM_BUILD_ROOT}/opt/oceanbase-diagnostic-tool/lib/site-packages
+\cp -rf $BUILD_DIR/SOURCES/resources ${RPM_BUILD_ROOT}/opt/oceanbase-diagnostic-tool/resources
+\cp -rf $BUILD_DIR/SOURCES/dependencies/bin ${RPM_BUILD_ROOT}/opt/oceanbase-diagnostic-tool/dependencies
+\cp -rf $BUILD_DIR/SOURCES/example ${RPM_BUILD_ROOT}/opt/oceanbase-diagnostic-tool/
+\cp -rf $BUILD_DIR/SOURCES/conf ${RPM_BUILD_ROOT}/opt/oceanbase-diagnostic-tool/conf
+\cp -rf $BUILD_DIR/SOURCES/init.sh ${RPM_BUILD_ROOT}/opt/oceanbase-diagnostic-tool/
+\cp -rf $BUILD_DIR/SOURCES/init_obdiag_cmd.sh ${RPM_BUILD_ROOT}/opt/oceanbase-diagnostic-tool/
+\cp -rf $BUILD_DIR/SOURCES/obdiag_backup.sh ${RPM_BUILD_ROOT}/opt/oceanbase-diagnostic-tool/
+\cp -rf $BUILD_DIR/SOURCES/plugins ${RPM_BUILD_ROOT}/opt/oceanbase-diagnostic-tool/
 
 
 %files
 %defattr(-,root,root,0777)
-/usr/local/oceanbase-diagnostic-tool/*
+/opt/oceanbase-diagnostic-tool/*
 
 %post
-chmod -R 755 /usr/local/oceanbase-diagnostic-tool/*
-chown -R root:root /usr/local/oceanbase-diagnostic-tool/*
-find /usr/local/oceanbase-diagnostic-tool/obdiag -type f -exec chmod 644 {} \;
-ln -sf /usr/local/oceanbase-diagnostic-tool/obdiag /usr/bin/obdiag
-chmod +x /usr/local/oceanbase-diagnostic-tool/obdiag
-cp -rf /usr/local/oceanbase-diagnostic-tool/init_obdiag_cmd.sh /etc/profile.d/obdiag.sh
-/usr/local/oceanbase-diagnostic-tool/obdiag_backup.sh
-/usr/local/oceanbase-diagnostic-tool/init.sh
+chmod -R 755 /opt/oceanbase-diagnostic-tool/*
+chown -R root:root /opt/oceanbase-diagnostic-tool/*
+find /opt/oceanbase-diagnostic-tool/obdiag -type f -exec chmod 644 {} \;
+ln -sf /opt/oceanbase-diagnostic-tool/obdiag /usr/bin/obdiag
+chmod +x /opt/oceanbase-diagnostic-tool/obdiag
+cp -rf /opt/oceanbase-diagnostic-tool/init_obdiag_cmd.sh /etc/profile.d/obdiag.sh
+/opt/oceanbase-diagnostic-tool/obdiag_backup.sh
+/opt/oceanbase-diagnostic-tool/init.sh
 echo -e 'Please execute the following command to init obdiag:\n'
-echo -e '\033[32m source /usr/local/oceanbase-diagnostic-tool/init.sh \n \033[0m'
+echo -e '\033[32m source /opt/oceanbase-diagnostic-tool/init.sh \n \033[0m'

